.htaccess

```diff
@@ -52,3 +52,5 @@ RewriteRule ^(.*)/(.*)/$ index.php?page=$1&section=$2 [L,QSA]
 RewriteRule ^(.*)/$ index.php?page=$1 [L]
 
 </IfModule>
+
+php_value xdebug.remote_host 192.168.1.78
```

Comment:
Why do you commit something like this?

Reply:
Sorry, it was supposed to be a private commit not a pull request...

Reply:
If you don't want it to be a pull request you can close the pull request.